test/E2E/HttpTrigger.Tests.ps1

```diff
@@ -14,66 +16,56 @@ Describe 'HttpTrigger Tests' {
         Get-Job -Name FuncJob -ErrorAction SilentlyContinue | Stop-Job | Remove-Job
     }
 
-    It "Test basic HttpTrigger function - Success" -TestCases @(
+    It "Test basic HttpTrigger function - Ok - $FunctionName" -TestCases @(
         @{ 
             FunctionName = 'TestBasicHttpTrigger'
-            InputNameData = 'Atlas'
-            ExpectedStatusCode = 200
             ExpectedContent = 'Hello Atlas'
         },
+        @{ 
+            FunctionName = 'TestBasicHttpTriggerWithTriggerMetadata'
+            ExpectedContent = 'Hello Atlas'
+        }
+    ) -Test {
+        param ($FunctionName, $ExpectedContent)
```

Comment:
Don't include `$FunctionName` in the param since it's not used in the test, only as part of the test name output

Reply:
fixed